quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/configuration/mappers/TracingPropertyMappers.java

```diff
@@ -47,6 +49,7 @@ public static PropertyMapper<?>[] getMappers() {
         return new PropertyMapper[]{
                 fromOption(TRACING_ENABLED)
                         .to("quarkus.otel.traces.enabled")
+                        .validator(TracingPropertyMappers::validateEnabled)
```

Comment:
Should we rather leverage `isEnabled` here so that the option is gone unless the Profile Feature is activated?